notebook/static/custom/custom.css

```diff
@@ -4,4 +4,66 @@ Placeholder for custom user CSS
 mainly to be overridden in profile/static/custom/custom.css
 
 This will always be an empty file in IPython
```

Comment:
This file needs to stay empty. If there is more CSS to include in this page, it should be added to  `static/tree/less/tree.less`.